suites/pacific/rados/tier-2_rados_test-osd-rebalance.yaml

```diff
@@ -147,3 +147,31 @@ tests:
               pg_num: 1
               disable_pg_autoscale: true
             # EC pool will be added later
+
+  # Below test is currently failing, BZ raised #2214864
+  # will be un-commented once fix is available
```

Comment:
Have a task in backlog, so that once the bug is addressed, this task can be picked up.